packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-## 44.17.0 - January 24, 2018
+## 44.18.0 - January 31, 2019
+
+### Fixed
+
+#### salesforcedx-vscode-core
+
+- Update deploy on save feature to exclusively trigger a code push or deploy when a file is saved ([PR #895](https://github.com/forcedotcom/salesforcedx-vscode/pull/895), [Issue #883](https://github.com/forcedotcom/salesforcedx-vscode/issues/883))
+
+#### salesforcedx-vscode-apex
+
+- Fix Apex LSP to update field references when they are renamed ([PR #905](https://github.com/forcedotcom/salesforcedx-vscode/pull/905))
```

Comment:
@praksb I might've over-simplified what the fix is so please let me know if we need to change it.

Reply:
@lcampos I would change this to something like "Fix in Apex LSP to update stale references in files with errors on an edit".